README.md

```diff
@@ -51,34 +47,74 @@ The following things are required to set up a SoftiCAR GitHub Runner:
 
 1. Clone this repository.
-1. Run the setup script, and install all components (i.e. Docker, Docker-Compose, GitHub CLI, and Sysbox):
+1. Use the setup script to install all required components (i.e. Docker, Docker-Compose, GitHub CLI, and Sysbox):
 
        ./setup install
 
-   - Reboot the VM afterwards.
-1. Log in to GitHub, using the PAT of the build-bot user:
-
-       gh auth login
-
-1. Copy `softicar-github-runner.env-example` to `/home/<user>/.softicar/softicar-github-runner.env`
-   - Read the comments in that file, and define the environment variables accordingly.
-1. Install and start the systemd service:
+   - Answer to various prompts.
+   - When prompted for a Personal Access Token (PAT), log in to the GitHub UI with the build-bot account.
+   - Head to `(Build-Bot User Profile)` / `Settings` / `Developer settings` / `Personal access tokens`.
+   - Create a PAT with settings and scopes as described in the terminal prompt.
+   - Copy and paste the PAT into the terminal prompt. **Watch out for leading and tailing whitespaces!**
+1. Reboot the VM afterwards.
+1. **Optionally,** copy `softicar-github-runner-service.env-example` to `/home/<user>/.softicar/softicar-github-runner-service.env`
+   - This is only necessary if default settings shall be overridden -- e.g. if a specific [GitHub Actions Runner](https://github.com/actions/runner) version shall be enforced.
+   - Refer to the comments in [softicar-github-runner-service.env-example](systemd-service/softicar-github-runner-service.env-example) and [softicar-github-runner-service-docker-compose.yml](systemd-service/softicar-github-runner-service-docker-compose.yml) for details.
+1. Install and start the systemd service: _(TODO rework this step when service installation gets migrated into setup.sh)_
 
-       ./service.sh install
-       ./service.sh start
+       ./control.sh install
+       ./control.sh start
 
    - Note that `install` will also _enable_ the service, i.e. set it to auto-start after reboots.
-1. Configure the cache proxy container:
-   1. **TODO** describe nexus setup and repository configuration
+1. Configure the _nexus_ container as a pull-though cache proxy for build dependencies:
+   1. Finish the first-time setup wizard of the _nexus_ container:
+      - Look up the default password of the `admin` user in `/var/lib/docker/volumes/nexus-data/_data/admin.password`
+      - Navigate to `http://<ip-of-vm>:8081`, and log in with `admin` / `(default-password)`
+      - Perform the displayed initialization steps. **Caution:** The default `admin.password` file will be deleted in the process.
+   1. Configure a **Docker image proxy:**
+      - Head to `Settings` / `Repository` / `Repositories`, click `Create repository`, and select `docker (proxy)`
+      - Enter:
+        - _Name:_ `docker-hub`
+        - _HTTP:_ Enabled, and enter `8123`
+        - _Allow anonymous docker pull:_ Enabled
+        - _Enable Docker V1 API:_ Enabled (**TODO** try without this)
```

Comment:
note: according to the docker docu, v2 does not support searching, if we need that, I cannot tell

Reply:
Good to know. We'll see about that in the final testing phase.